2024-12-01 02:32:29,673 - ERROR - URL Error http://s1.voscast.com:9038/mountpoint3: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:32:30,015 - ERROR - URL Error http://ourfile.info/akaram.mp3: HTTPConnectionPool(host='ourfile.info', port=80): Max retries exceeded with url: /akaram.mp3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bd17f10>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:32:45,031 - ERROR - URL Error http://69.46.24.226:7188/stream: HTTPConnectionPool(host='69.46.24.226', port=7188): Max retries exceeded with url: /stream (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd17dc0>, 'Connection to 69.46.24.226 timed out. (connect timeout=15)'))
2024-12-01 02:32:45,060 - ERROR - URL Error http://s7.viastreaming.net:7920/;stream.mp3: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:32:45,283 - ERROR - URL Error http://s2.voscast.com:9370/;stream.mp3: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:33:00,299 - ERROR - URL Error http://37.187.79.93:9350/stream: HTTPConnectionPool(host='37.187.79.93', port=9350): Max retries exceeded with url: /stream (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd570>, 'Connection to 37.187.79.93 timed out. (connect timeout=15)'))
2024-12-01 02:33:01,474 - ERROR - URL Error http://s6.voscast.com:10014/;stream.mp3: HTTPConnectionPool(host='s6.voscast.com', port=10014): Max retries exceeded with url: /;stream.mp3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bd17970>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:33:16,960 - ERROR - URL Error http://live.cmr24.net:8081/CMR/TamilHD-LQ/icecast.audio: HTTPConnectionPool(host='live.cmr24.net', port=8081): Max retries exceeded with url: /CMR/TamilHD-LQ/icecast.audio (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd8d0>, 'Connection to live.cmr24.net timed out. (connect timeout=15)'))
2024-12-01 02:33:32,352 - ERROR - URL Error http://tstv-stream.tsm.utexas.edu/hls/livestream_hi/index.m3u8: HTTPConnectionPool(host='tstv-stream.tsm.utexas.edu', port=80): Max retries exceeded with url: /hls/livestream_hi/index.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd0f0>, 'Connection to tstv-stream.tsm.utexas.edu timed out. (connect timeout=15)'))
2024-12-01 02:33:41,606 - ERROR - URL Error https://5928f90056f31.streamlock.net/ibc-test/smil:ibc_tamiltv.smil/chunklist.m3u8: HTTPSConnectionPool(host='5928f90056f31.streamlock.net', port=443): Max retries exceeded with url: /ibc-test/smil:ibc_tamiltv.smil/chunklist.m3u8 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
2024-12-01 02:34:11,653 - ERROR - URL Error http://live.tamilvision.tv:8081/TVI/SD/chunks.m3u8: HTTPConnectionPool(host='live.tamilvision.tv', port=8081): Max retries exceeded with url: /TVI/SD/chunks.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccbb0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
2024-12-01 02:34:41,685 - ERROR - URL Error http://live.tamilvision.tv:8081/TVI/HD/playlist.m3u8: HTTPConnectionPool(host='live.tamilvision.tv', port=8081): Max retries exceeded with url: /TVI/HD/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcda20>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
2024-12-01 02:34:41,973 - ERROR - URL Error https://mediacpstreamchile.com:1936/hiperconectados/hiperconectados/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /hiperconectados/hiperconectados/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17970>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:34:42,378 - ERROR - URL Error https://live1.presstv.ir/live/hispan.m3u8: HTTPSConnectionPool(host='live1.presstv.ir', port=443): Max retries exceeded with url: /live/hispan.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd16a10>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:34:58,482 - ERROR - URL Error http://hitmusic.hu/hitmusic.m3u8: HTTPConnectionPool(host='hitmusic.hu', port=80): Max retries exceeded with url: /hitmusic.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd16650>, 'Connection to hitmusic.hu timed out. (connect timeout=15)'))
2024-12-01 02:34:58,667 - ERROR - URL Error http://85.25.217.22:8310/stream: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:34:59,527 - ERROR - URL Error http://cm.hostlagarto.com:8081/hits360tv/hits360HD.myStream/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='cm.hostlagarto.com', port=8081): Max retries exceeded with url: /hits360tv/hits360HD.myStream/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc400>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:35:09,544 - ERROR - URL Error http://astro1.rastream.com/hitz: HTTPConnectionPool(host='astro1.rastream.com', port=80): Max retries exceeded with url: /hitz (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccbb0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:35:19,559 - ERROR - URL Error http://astro1.rastream.com/amp-hitz-kk: HTTPConnectionPool(host='astro1.rastream.com', port=80): Max retries exceeded with url: /amp-hitz-kk (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd3c0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:35:29,572 - ERROR - URL Error http://astro1.rastream.com/amp-hitz-kch: HTTPConnectionPool(host='astro1.rastream.com', port=80): Max retries exceeded with url: /amp-hitz-kch (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc5b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:35:40,748 - ERROR - URL Error http://mediaprima.rastream.com/mediaprima-hotfm?awparams=companionads%3Atrue%3Btags%3Aradioactive%3Bstationid%3Amediaprima-hotfm&playerid=Hot%20FM_web&authtoken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJvaWQiOiJsYXlsaW8iLCJpYXQiOjE0NzkxMDQyNzY1MDUsImV4cCI6M: HTTPConnectionPool(host='mediaprima.rastream.com', port=80): Max retries exceeded with url: /mediaprima-hotfm?awparams=companionads%3Atrue%3Btags%3Aradioactive%3Bstationid%3Amediaprima-hotfm&playerid=Hot%20FM_web&authtoken=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJvaWQiOiJsYXlsaW8iLCJpYXQiOjE0NzkxMDQyNzY1MDUsImV4cCI6M (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccbe0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:35:40,910 - ERROR - URL Error https://tv.portalexpress.es:3124/live/hchoperutvlive.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='tv.portalexpress.es', port=3124): Max retries exceeded with url: /live/hchoperutvlive.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccc40>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:35:42,570 - ERROR - URL Error http://s1.sonicabroadcast.com:7005/;stream/1: HTTPConnectionPool(host='s1.sonicabroadcast.com', port=7005): Max retries exceeded with url: /;stream/1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc7f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:35:53,496 - ERROR - URL Error http://astro4.rastream.com/india: HTTPConnectionPool(host='astro4.rastream.com', port=80): Max retries exceeded with url: /india (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce3b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:36:10,910 - ERROR - URL Error https://5fa5de1a545ae.streamlock.net:443/sisjalisciense/sisjalisciense/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='5fa5de1a545ae.streamlock.net', port=443): Max retries exceeded with url: /sisjalisciense/sisjalisciense/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd174c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:36:11,137 - ERROR - URL Error https://emisoras.hn:8081/jbn/index.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='emisoras.hn', port=8081): Max retries exceeded with url: /jbn/index.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:36:11,572 - ERROR - URL Error https://servilive.com:3149/live/jn19tvlive.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='servilive.com', port=3149): Max retries exceeded with url: /live/jn19tvlive.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd180>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:36:26,711 - ERROR - URL Error https://videostreaming.cloudserverlatam.com:443/Kids90/Kids90/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='videostreaming.cloudserverlatam.com', port=443): Max retries exceeded with url: /Kids90/Kids90/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccf70>, 'Connection to videostreaming.cloudserverlatam.com timed out. (connect timeout=15)'))
2024-12-01 02:36:26,760 - ERROR - URL Error http://flashcloud.mediacdn.com/live/kontratv/chunklist_w1477516246.m3u8: HTTPConnectionPool(host='flashcloud.mediacdn.com', port=80): Max retries exceeded with url: /live/kontratv/chunklist_w1477516246.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:36:36,781 - ERROR - URL Error https://mediaprima.rastream.com/mediaprima-koolfm: HTTPSConnectionPool(host='mediaprima.rastream.com', port=443): Max retries exceeded with url: /mediaprima-koolfm (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcccd0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:36:53,748 - ERROR - URL Error http://astro1.rastream.com/lite: HTTPConnectionPool(host='astro1.rastream.com', port=80): Max retries exceeded with url: /lite (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcce20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:36:55,035 - ERROR - URL Error http://bamus-spa-roku.amagi.tv/playlist.m3u8?ROGERIOTORRES: HTTPConnectionPool(host='bamus-spa-roku.amagi.tv', port=80): Max retries exceeded with url: /playlist.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc5e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:36:55,094 - ERROR - URL Error https://bamus-eng-roku.amagi.tv/playlist.m3u8: HTTPSConnectionPool(host='bamus-eng-roku.amagi.tv', port=443): Max retries exceeded with url: /playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:37:10,722 - ERROR - URL Error https://cdn4.hostlagarto.com:8081/static/lunatv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdn4.hostlagarto.com', port=8081): Max retries exceeded with url: /static/lunatv/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce5c0>, 'Connection to cdn4.hostlagarto.com timed out. (connect timeout=15)'))
2024-12-01 02:37:27,236 - ERROR - URL Error https://videostreaming.cloudserverlatam.com:443/8040/8040/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='videostreaming.cloudserverlatam.com', port=443): Max retries exceeded with url: /8040/8040/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdb70>, 'Connection to videostreaming.cloudserverlatam.com timed out. (connect timeout=15)'))
2024-12-01 02:37:29,320 - ERROR - URL Error http://stream3.prostudionetwork.com:1943/megavision/MV/chunklist_w1925216201.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='stream3.prostudionetwork.com', port=1943): Max retries exceeded with url: /megavision/MV/chunklist_w1925216201.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdcc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:37:29,375 - ERROR - URL Error https://livefocamundo.com:8081/megavisiontv/index.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='livefocamundo.com', port=8081): Max retries exceeded with url: /megavisiontv/index.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17e80>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:37:39,390 - ERROR - URL Error https://astro3.rastream.com/melody: HTTPSConnectionPool(host='astro3.rastream.com', port=443): Max retries exceeded with url: /melody (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17fa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:37:56,189 - ERROR - URL Error http://wms.tecnoxia.com:1935/8006/8006/playlist.m3u8: HTTPConnectionPool(host='wms.tecnoxia.com', port=1935): Max retries exceeded with url: /8006/8006/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd17370>, 'Connection to wms.tecnoxia.com timed out. (connect timeout=15)'))
2024-12-01 02:38:08,045 - ERROR - URL Error https://astro2.rastream.com/mix: HTTPSConnectionPool(host='astro2.rastream.com', port=443): Max retries exceeded with url: /mix (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce290>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:38:13,726 - ERROR - URL Error https://mmatv-live.localnowlive.com/v1/manifest/385c85a93929f94966d0fb186fc33b431e6f1eb9/DSP_61327d317542b31350435482_mmatv-live/a504c826-6950-4b5b-a518-171d9aef07ee/4.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='mmatv-live.localnowlive.com', port=443): Max retries exceeded with url: /v1/manifest/385c85a93929f94966d0fb186fc33b431e6f1eb9/DSP_61327d317542b31350435482_mmatv-live/a504c826-6950-4b5b-a518-171d9aef07ee/4.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc580>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:38:18,081 - ERROR - URL Error http://ventdelnord.tv:8080/mola/directe.m3u8?PlaylistM3UCL: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:38:22,501 - ERROR - URL Error http://ventdelnord.tv:8080/mola/directe.m3u8: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:38:22,975 - ERROR - URL Error http://momentog-crtvg.flumotion.com/playlist.m3u8: HTTPConnectionPool(host='momentog-crtvg.flumotion.com', port=80): Max retries exceeded with url: /playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd0c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:38:22,983 - ERROR - URL Error http://momentog-crtvg.flumotion.com/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='momentog-crtvg.flumotion.com', port=80): Max retries exceeded with url: /playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccbe0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:38:23,566 - ERROR - URL Error http://vs8.live.opencaster.com/20100152275/jcpstream/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='vs8.live.opencaster.com', port=80): Max retries exceeded with url: /20100152275/jcpstream/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccfd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:38:40,264 - ERROR - URL Error https://panel.dattalive.com:443/8250/8250/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='panel.dattalive.com', port=443): Max retries exceeded with url: /8250/8250/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc1c0>, 'Connection to panel.dattalive.com timed out. (connect timeout=15)'))
2024-12-01 02:39:06,828 - ERROR - URL Error http://panel.dattalive.com:1935/8250/8250/playlist.m3u8: HTTPConnectionPool(host='panel.dattalive.com', port=1935): Read timed out. (read timeout=15)
2024-12-01 02:39:21,920 - ERROR - URL Error https://videostreaming.cloudserverlatam.com:443/8078/8078/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='videostreaming.cloudserverlatam.com', port=443): Max retries exceeded with url: /8078/8078/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17af0>, 'Connection to videostreaming.cloudserverlatam.com timed out. (connect timeout=15)'))
2024-12-01 02:39:35,119 - ERROR - URL Error https://astro1.rastream.com/myfm: HTTPSConnectionPool(host='astro1.rastream.com', port=443): Max retries exceeded with url: /myfm (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce7d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:39:45,139 - ERROR - URL Error http://astro1.rastream.com/amp-myfm-kk: HTTPConnectionPool(host='astro1.rastream.com', port=80): Max retries exceeded with url: /amp-myfm-kk (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce3b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:39:55,160 - ERROR - URL Error http://astro1.rastream.com/amp-myfm-kch: HTTPConnectionPool(host='astro1.rastream.com', port=80): Max retries exceeded with url: /amp-myfm-kch (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce560>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:39:55,755 - ERROR - URL Error http://audio8.cmaudioevideo.com:8038/stream: HTTPConnectionPool(host='audio8.cmaudioevideo.com', port=8038): Max retries exceeded with url: /stream (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce3e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:39:56,146 - ERROR - URL Error https://streamconex.com:1936/n24tv/n24tv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='streamconex.com', port=1936): Max retries exceeded with url: /n24tv/n24tv/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce650>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:39:57,766 - ERROR - URL Error https://mediacpstreamchile.com:1936/nativa3/nativa3/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /nativa3/nativa3/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcca30>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:40:16,427 - ERROR - URL Error http://live.prd.go.th:1935/live/ch1_L.sdp/chunklist_w1249643471.m3u8: HTTPConnectionPool(host='live.prd.go.th', port=1935): Max retries exceeded with url: /live/ch1_L.sdp/chunklist_w1249643471.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcda20>, 'Connection to live.prd.go.th timed out. (connect timeout=15)'))
2024-12-01 02:40:37,054 - ERROR - URL Error http://amaru.dyndns.org:8870/0.m3u8: HTTPConnectionPool(host='amaru.dyndns.org', port=8870): Read timed out. (read timeout=15)
2024-12-01 02:40:43,569 - ERROR - URL Error https://partne.cdn.mangomolo.com/omsport/smil:omsport.stream.smil/playlist.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='partne.cdn.mangomolo.com', port=443): Max retries exceeded with url: /omsport/smil:omsport.stream.smil/playlist.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17550>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:40:43,956 - ERROR - URL Error http://partne.cdn.mangomolo.com/omantv/smil:omantv.stream.smil/playlist.m3u8: HTTPConnectionPool(host='partne.cdn.mangomolo.com', port=80): Max retries exceeded with url: /omantv/smil:omantv.stream.smil/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bd175b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:40:56,321 - ERROR - URL Error https://mediaprima.rastream.com/mediaprima-onefm: HTTPSConnectionPool(host='mediaprima.rastream.com', port=443): Max retries exceeded with url: /mediaprima-onefm (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17af0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:40:57,752 - ERROR - URL Error https://cdn2.zencast.tv:30443/orbe/orbe21smil/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdn2.zencast.tv', port=30443): Max retries exceeded with url: /orbe/orbe21smil/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd16680>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:40:58,256 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8076/8076/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8076/8076/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:40:59,704 - ERROR - URL Error http://xrxs.net/video/live-p12baya-2328.m3u8: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:41:15,544 - ERROR - URL Error https://5fb24b460df87.streamlock.net/live-cont.ar/pakapaka/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='5fb24b460df87.streamlock.net', port=443): Max retries exceeded with url: /live-cont.ar/pakapaka/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc2e0>, 'Connection to 5fb24b460df87.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:41:31,442 - ERROR - URL Error http://193.147.254.64:1935/realizacion1/realizacion1/playlist.m3u8: HTTPConnectionPool(host='193.147.254.64', port=1935): Max retries exceeded with url: /realizacion1/realizacion1/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdf30>, 'Connection to 193.147.254.64 timed out. (connect timeout=15)'))
2024-12-01 02:41:46,458 - ERROR - URL Error http://193.147.254.64:1935/realizacion2/realizacion2/playlist.m3u8: HTTPConnectionPool(host='193.147.254.64', port=1935): Max retries exceeded with url: /realizacion2/realizacion2/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd240>, 'Connection to 193.147.254.64 timed out. (connect timeout=15)'))
2024-12-01 02:42:01,474 - ERROR - URL Error http://193.147.254.64:1935/realizacion3/realizacion3/playlist.m3u8: HTTPConnectionPool(host='193.147.254.64', port=1935): Max retries exceeded with url: /realizacion3/realizacion3/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbce6e0>, 'Connection to 193.147.254.64 timed out. (connect timeout=15)'))
2024-12-01 02:42:16,491 - ERROR - URL Error http://193.147.254.64:1935/realizacion4/realizacion4/playlist.m3u8: HTTPConnectionPool(host='193.147.254.64', port=1935): Max retries exceeded with url: /realizacion4/realizacion4/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd090>, 'Connection to 193.147.254.64 timed out. (connect timeout=15)'))
2024-12-01 02:42:36,220 - ERROR - URL Error http://149.56.17.92:1935/portalfoxmix/_definst_/portalfoxmix/playlist.m3u8: HTTPConnectionPool(host='149.56.17.92', port=1935): Max retries exceeded with url: /portalfoxmix/_definst_/portalfoxmix/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd17a00>, 'Connection to 149.56.17.92 timed out. (connect timeout=15)'))
2024-12-01 02:42:38,972 - ERROR - URL Error http://213.13.26.11:1935/live/portocanal/playlist.m3u8?IMDSFULL: HTTPConnectionPool(host='213.13.26.11', port=1935): Max retries exceeded with url: /live/portocanal/playlist.m3u8?IMDSFULL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd270>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:42:39,087 - ERROR - URL Error http://213.13.26.11:1935/live/portocanal/KodiPortugal.pt.m3u8: HTTPConnectionPool(host='213.13.26.11', port=1935): Max retries exceeded with url: /live/portocanal/KodiPortugal.pt.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce4a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:42:39,200 - ERROR - URL Error http://213.13.26.11:1935/live/portocanal/livestream.m3u8: HTTPConnectionPool(host='213.13.26.11', port=1935): Max retries exceeded with url: /live/portocanal/livestream.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccbe0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:42:39,312 - ERROR - URL Error http://213.13.26.11:1935/live/portocanal/KodiPortugal.pt.m3u8?ROGERIOTORRES: HTTPConnectionPool(host='213.13.26.11', port=1935): Max retries exceeded with url: /live/portocanal/KodiPortugal.pt.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc940>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:42:52,535 - ERROR - URL Error https://live1.presstv.ir/live/presstv.m3u8: HTTPSConnectionPool(host='live1.presstv.ir', port=443): Max retries exceeded with url: /live/presstv.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd173d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:43:07,580 - ERROR - URL Error https://5ca39be538307.streamlock.net/telemetrika/smil:telemetrika.smil/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='5ca39be538307.streamlock.net', port=443): Max retries exceeded with url: /telemetrika/smil:telemetrika.smil/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17dc0>, 'Connection to 5ca39be538307.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:43:08,190 - ERROR - URL Error http://213.13.26.11:1935/live/portocanal/magellan.m3u8: HTTPConnectionPool(host='213.13.26.11', port=1935): Max retries exceeded with url: /live/portocanal/magellan.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdb40>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:43:09,979 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8012/8012/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8012/8012/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:43:10,313 - ERROR - URL Error https://tv7api2.tv.init7.net/api/live/?channel=cb35debd-e476-4bd0-a14e-bf30b6e6e7db: HTTPSConnectionPool(host='tv7api2.tv.init7.net', port=443): Max retries exceeded with url: /api/live/?channel=cb35debd-e476-4bd0-a14e-bf30b6e6e7db (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbceb30>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:43:26,115 - ERROR - URL Error https://videostreaming.cloudserverlatam.com:443/8060/8060/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='videostreaming.cloudserverlatam.com', port=443): Max retries exceeded with url: /8060/8060/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc910>, 'Connection to videostreaming.cloudserverlatam.com timed out. (connect timeout=15)'))
2024-12-01 02:43:36,131 - ERROR - URL Error https://astro3.rastream.com/raaga: HTTPSConnectionPool(host='astro3.rastream.com', port=443): Max retries exceeded with url: /raaga (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce320>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:43:37,871 - ERROR - URL Error https://593b04c4c5670.streamlock.net/8020/8020/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8020/8020/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:43:39,733 - ERROR - URL Error https://59537faa0729a.streamlock.net:443/radioblu/radioblu/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59537faa0729a.streamlock.net', port=443): Max retries exceeded with url: /radioblu/radioblu/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc940>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:43:40,691 - ERROR - URL Error https://593b04c4c5670.streamlock.net/8142/8142/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8142/8142/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:43:42,567 - ERROR - URL Error https://media1.mcr.iol.pt/livefm/cidade/livefm/cidade_128k/chunks.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /livefm/cidade/livefm/cidade_128k/chunks.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc850>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:43:42,977 - ERROR - URL Error https://media1.mcr.iol.pt/wrcidade/cidhiphop.stream/playlist.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /wrcidade/cidhiphop.stream/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccb80>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:43:43,257 - ERROR - URL Error https://media1.mcr.iol.pt/wrcidade/cidlatina.stream/playlist.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /wrcidade/cidlatina.stream/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdde0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:43:43,518 - ERROR - URL Error http://streamserver2.somosdigital.net:2199/system/images/tunein-pls.png: HTTPConnectionPool(host='streamserver2.somosdigital.net', port=2199): Max retries exceeded with url: /system/images/tunein-pls.png (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdf90>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:43:44,359 - ERROR - URL Error https://media1.mcr.iol.pt/livefm/comercial/livefm/comercial_128k/chunks.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /livefm/comercial/livefm/comercial_128k/chunks.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:01,191 - ERROR - URL Error https://videostreaming.cloudserverlatam.com:443/8052/8052/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='videostreaming.cloudserverlatam.com', port=443): Max retries exceeded with url: /8052/8052/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bd178b0>, 'Connection to videostreaming.cloudserverlatam.com timed out. (connect timeout=15)'))
2024-12-01 02:44:04,014 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8034/8034/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8034/8034/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:44:04,941 - ERROR - URL Error https://media1.mcr.iol.pt/livefm/m80/livefm/m80_128k/chunks.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /livefm/m80/livefm/m80_128k/chunks.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdde0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:05,615 - ERROR - URL Error https://radiomarcabcnmedia.dns-lcinternet.com:9000/stream: HTTPSConnectionPool(host='radiomarcabcnmedia.dns-lcinternet.com', port=9000): Max retries exceeded with url: /stream (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))
2024-12-01 02:44:06,701 - ERROR - URL Error http://94.23.87.98:8065/;&type=mp3: HTTPConnectionPool(host='94.23.87.98', port=8065): Max retries exceeded with url: /;&type=mp3 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdc90>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:44:06,738 - ERROR - URL Error https://21313.live.streamtheworld.com/MEGA_HITSAAC: HTTPSConnectionPool(host='21313.live.streamtheworld.com', port=443): Max retries exceeded with url: /MEGA_HITSAAC (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd750>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:07,778 - ERROR - URL Error http://centova.radio.com.pt:9424/;: HTTPConnectionPool(host='centova.radio.com.pt', port=9424): Max retries exceeded with url: /; (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcde70>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:44:22,972 - ERROR - URL Error http://91.121.156.27:8013/stream: HTTPConnectionPool(host='91.121.156.27', port=8013): Max retries exceeded with url: /stream (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc790>, 'Connection to 91.121.156.27 timed out. (connect timeout=15)'))
2024-12-01 02:44:26,338 - ERROR - URL Error https://proic1.evspt.com/oxigenio_aac: HTTPSConnectionPool(host='proic1.evspt.com', port=443): Max retries exceeded with url: /oxigenio_aac (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))
2024-12-01 02:44:27,562 - ERROR - URL Error https://mediacpstreamchile.com:1936/radiopolar2/radiopolar2/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /radiopolar2/radiopolar2/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdc30>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:29,313 - ERROR - URL Error https://streaming5.globalhostla.com/rtplive/puntofmcl/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='streaming5.globalhostla.com', port=443): Max retries exceeded with url: /rtplive/puntofmcl/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))
2024-12-01 02:44:30,593 - ERROR - URL Error https://proic1.evspt.com/radar_aac: HTTPSConnectionPool(host='proic1.evspt.com', port=443): Max retries exceeded with url: /radar_aac (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))
2024-12-01 02:44:34,169 - ERROR - URL Error https://21313.live.streamtheworld.com/RFMAAC: HTTPSConnectionPool(host='21313.live.streamtheworld.com', port=443): Max retries exceeded with url: /RFMAAC (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcead0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:34,355 - ERROR - URL Error http://85.25.217.22:8210/stream: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:44:36,626 - ERROR - URL Error https://centova.radios.pt/proxy/435?mp=/stream: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:44:53,054 - ERROR - URL Error https://ssl.evsportugal.net:7166/stream: HTTPSConnectionPool(host='ssl.evsportugal.net', port=7166): Max retries exceeded with url: /stream (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd210>, 'Connection to ssl.evsportugal.net timed out. (connect timeout=15)'))
2024-12-01 02:44:53,172 - ERROR - URL Error https://media1.mcr.iol.pt/livefm/smooth/livefm/smooth_128k/chunks.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /livefm/smooth/livefm/smooth_128k/chunks.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccf10>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:54,912 - ERROR - URL Error https://mediacpstreamchile.com:1936/8118/8118/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /8118/8118/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc130>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:44:57,059 - ERROR - URL Error https://centova.radio.com.pt/proxy/558?mp=/stream: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:45:01,364 - ERROR - URL Error https://Stmv1.zcastbr.com/julio4589/julio4589/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='stmv1.zcastbr.com', port=443): Max retries exceeded with url: /julio4589/julio4589/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcce50>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:45:02,238 - ERROR - URL Error https://59537faa0729a.streamlock.net:443/rbdvideo/rbdvideo/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59537faa0729a.streamlock.net', port=443): Max retries exceeded with url: /rbdvideo/rbdvideo/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdcf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:45:17,273 - ERROR - URL Error https://59ce1298bfb98.streamlock.net/rcctv02/rcctv02/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59ce1298bfb98.streamlock.net', port=443): Max retries exceeded with url: /rcctv02/rcctv02/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc1c0>, 'Connection to 59ce1298bfb98.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:45:32,822 - ERROR - URL Error http://190.103.183.24:1935/ReadyTV/ReadyHD/chunklist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='190.103.183.24', port=1935): Max retries exceeded with url: /ReadyTV/ReadyHD/chunklist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcceb0>, 'Connection to 190.103.183.24 timed out. (connect timeout=15)'))
2024-12-01 02:45:33,451 - ERROR - URL Error https://rede-muhler-recordnews-1-br.samsung.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='rede-muhler-recordnews-1-br.samsung.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd16470>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:45:35,725 - ERROR - URL Error https://s1.tvdatta.com:3707/live/redtvelive.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='s1.tvdatta.com', port=3707): Max retries exceeded with url: /live/redtvelive.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17460>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:45:39,891 - ERROR - URL Error http://rtmp.cdn.upx.net.br/00084/myStream.sdp/playlist.m3u8: HTTPConnectionPool(host='rtmp.cdn.upx.net.br', port=80): Max retries exceeded with url: /00084/myStream.sdp/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd960>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:45:50,475 - ERROR - URL Error http://v2.tustreaming.cl/restaurandovidastv/index.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='v2.tustreaming.cl', port=80): Max retries exceeded with url: /restaurandovidastv/index.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdc90>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:46:07,931 - ERROR - URL Error http://89.185.253.55/retrotv/retrotvHQ1/playlist.m3u8: HTTPConnectionPool(host='89.185.253.55', port=80): Max retries exceeded with url: /retrotv/retrotvHQ1/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc400>, 'Connection to 89.185.253.55 timed out. (connect timeout=15)'))
2024-12-01 02:46:08,872 - ERROR - URL Error https://reuters-reuters-1-eu.rakuten.wurl.com/manifest/playlist.m3u8: HTTPSConnectionPool(host='reuters-reuters-1-eu.rakuten.wurl.com', port=443): Max retries exceeded with url: /manifest/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:46:09,470 - ERROR - URL Error http://evo.eltelon.com:1935/live/rocco-tv/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='evo.eltelon.com', port=1935): Max retries exceeded with url: /live/rocco-tv/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbccdf0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:46:24,798 - ERROR - URL Error http://fms-https1.mediadirect.ro/live3/_definst_/rocktv/chunklist_b1600000.m3u8: HTTPConnectionPool(host='fms-https1.mediadirect.ro', port=80): Max retries exceeded with url: /live3/_definst_/rocktv/chunklist_b1600000.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcebc0>, 'Connection to fms-https1.mediadirect.ro timed out. (connect timeout=15)'))
2024-12-01 02:46:40,965 - ERROR - URL Error http://136.243.3.70:1935/RtpBolivia/RtpBolivia/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='136.243.3.70', port=1935): Max retries exceeded with url: /RtpBolivia/RtpBolivia/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd17a00>, 'Connection to 136.243.3.70 timed out. (connect timeout=15)'))
2024-12-01 02:46:57,367 - ERROR - URL Error http://wms.tecnoxia.com:1935/rytqrolive/rytqrolive/master.m3u8: HTTPConnectionPool(host='wms.tecnoxia.com', port=1935): Max retries exceeded with url: /rytqrolive/rytqrolive/master.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcca60>, 'Connection to wms.tecnoxia.com timed out. (connect timeout=15)'))
2024-12-01 02:46:58,918 - ERROR - URL Error http://192.111.140.6:8142: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:47:01,022 - ERROR - URL Error http://servidor26.brlogic.com:8084/live?1541552495852: HTTPConnectionPool(host='servidor26.brlogic.com', port=8084): Max retries exceeded with url: /live?1541552495852 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce290>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:47:17,035 - ERROR - URL Error https://evpp.mm.uol.com.br/ne10/ne10.smil/playlist.m3u8: HTTPSConnectionPool(host='evpp.mm.uol.com.br', port=443): Max retries exceeded with url: /ne10/ne10.smil/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd810>, 'Connection to evpp.mm.uol.com.br timed out. (connect timeout=15)'))
2024-12-01 02:47:17,197 - ERROR - URL Error https://livefocamundo.com:8081/tvguajaraonline2/tracks-v1a1/mono.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='livefocamundo.com', port=8081): Max retries exceeded with url: /tvguajaraonline2/tracks-v1a1/mono.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc130>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:47:17,734 - ERROR - URL Error https://cdn-cdn-iguacu.ciclano.io:1443/cdn-iguacu/cdn-iguacu/playlist.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='cdn-cdn-iguacu.ciclano.io', port=1443): Max retries exceeded with url: /cdn-iguacu/cdn-iguacu/playlist.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbced70>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:47:34,879 - ERROR - URL Error http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/chunklist_w278545363_b1128000_slpl.m3u8: HTTPConnectionPool(host='stream.sferatv.pl', port=1935): Read timed out. (read timeout=15)
2024-12-01 02:47:51,578 - ERROR - URL Error http://5953e7f9ae845.streamlock.net/2502/2502/chunklist.m3u8: HTTPConnectionPool(host='5953e7f9ae845.streamlock.net', port=80): Max retries exceeded with url: /2502/2502/chunklist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd165f0>, 'Connection to 5953e7f9ae845.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:48:02,063 - ERROR - URL Error http://astro2.rastream.com/sinar: HTTPConnectionPool(host='astro2.rastream.com', port=80): Max retries exceeded with url: /sinar (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbced40>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:48:02,903 - ERROR - URL Error http://213.13.26.11:1935/live/sobrenaturaltv/livestream.m3u8: HTTPConnectionPool(host='213.13.26.11', port=1935): Max retries exceeded with url: /live/sobrenaturaltv/livestream.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdf30>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:48:18,281 - ERROR - URL Error https://59ce1298bfb98.streamlock.net/somosdeleste/somosdeleste/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59ce1298bfb98.streamlock.net', port=443): Max retries exceeded with url: /somosdeleste/somosdeleste/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccc70>, 'Connection to 59ce1298bfb98.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:48:18,386 - ERROR - URL Error https://mediacpstreamchile.com:1936/sonoonda/sonoonda/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /sonoonda/sonoonda/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbceb00>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:48:20,219 - ERROR - URL Error https://eu8.fastcast4u.com/proxy/spectrumcds2?mp=/1: ('Connection aborted.', BadStatusLine('RPC-ERR authentication failed\n'))
2024-12-01 02:48:21,560 - ERROR - URL Error http://b5.vc/live/5sVAyGZp/phantomiptv/72330.m3u8: HTTPConnectionPool(host='b5.vc', port=80): Max retries exceeded with url: /live/5sVAyGZp/phantomiptv/72330.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc790>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:48:37,403 - ERROR - URL Error https://stv.janus.cl/playlist/stream.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='stv.janus.cl', port=443): Max retries exceeded with url: /playlist/stream.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce050>, 'Connection to stv.janus.cl timed out. (connect timeout=15)'))
2024-12-01 02:48:52,444 - ERROR - URL Error https://59ce1298bfb98.streamlock.net/sucesotv/sucesotv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59ce1298bfb98.streamlock.net', port=443): Max retries exceeded with url: /sucesotv/sucesotv/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bd166b0>, 'Connection to 59ce1298bfb98.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:49:07,665 - ERROR - URL Error http://live.cdn.upx.com/00046/myStream.sdp/playlist.m3u8: HTTPConnectionPool(host='live.cdn.upx.com', port=80): Max retries exceeded with url: /00046/myStream.sdp/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd179a0>, 'Connection to live.cdn.upx.com timed out. (connect timeout=15)'))
2024-12-01 02:49:17,715 - ERROR - URL Error https://astro4.rastream.com/bayu.web: HTTPSConnectionPool(host='astro4.rastream.com', port=443): Max retries exceeded with url: /bayu.web (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc430>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:49:27,726 - ERROR - URL Error http://astro4.rastream.com/rock: HTTPConnectionPool(host='astro4.rastream.com', port=80): Max retries exceeded with url: /rock (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce3b0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:49:37,740 - ERROR - URL Error http://astro4.rastream.com/gold: HTTPConnectionPool(host='astro4.rastream.com', port=80): Max retries exceeded with url: /gold (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc8e0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:49:47,754 - ERROR - URL Error http://astro4.rastream.com/jazz: HTTPConnectionPool(host='astro4.rastream.com', port=80): Max retries exceeded with url: /jazz (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce920>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:49:57,778 - ERROR - URL Error https://astro4.rastream.com/kenyalang.web: HTTPSConnectionPool(host='astro4.rastream.com', port=443): Max retries exceeded with url: /kenyalang.web (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcc100>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:50:07,795 - ERROR - URL Error http://astro4.rastream.com/opus: HTTPConnectionPool(host='astro4.rastream.com', port=80): Max retries exceeded with url: /opus (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcce20>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:50:17,816 - ERROR - URL Error https://astro4.rastream.com/osai: HTTPSConnectionPool(host='astro4.rastream.com', port=443): Max retries exceeded with url: /osai (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce920>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))
2024-12-01 02:50:20,064 - ERROR - URL Error https://play.amelbasoluciones.co:3971/live/tdicolombiatvlive.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='play.amelbasoluciones.co', port=3971): Max retries exceeded with url: /live/tdicolombiatvlive.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce2f0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:50:23,286 - ERROR - URL Error http://190.107.176.55:1935/live/tec-tv/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='190.107.176.55', port=1935): Max retries exceeded with url: /live/tec-tv/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcee30>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:50:38,622 - ERROR - URL Error https://stmvideo2.livecastv.com/telestrella/telestrella/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='stmvideo2.livecastv.com', port=443): Max retries exceeded with url: /telestrella/telestrella/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcee90>, 'Connection to stmvideo2.livecastv.com timed out. (connect timeout=15)'))
2024-12-01 02:50:55,899 - ERROR - URL Error https://cdn4.hostlagarto.com:8081/static/telecanal12/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdn4.hostlagarto.com', port=8081): Max retries exceeded with url: /static/telecanal12/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd930>, 'Connection to cdn4.hostlagarto.com timed out. (connect timeout=15)'))
2024-12-01 02:50:57,627 - ERROR - URL Error https://cm.hostlagarto.com:4445/Telecibao/Telecibao/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cm.hostlagarto.com', port=4445): Max retries exceeded with url: /Telecibao/Telecibao/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf580>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:51:28,240 - ERROR - URL Error https://mediacp.hostradios.com.ar:19360/telediario/telediario.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacp.hostradios.com.ar', port=19360): Max retries exceeded with url: /telediario/telediario.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd16860>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
2024-12-01 02:51:28,714 - ERROR - URL Error plugin://plugin.video.youtube/play/?video_id=B8Xd75Mvup8&list=UUy-r-BQ5BQRU6rzGS73-WBQ: No connection adapters were found for 'plugin://plugin.video.youtube/play/?video_id=B8Xd75Mvup8&list=UUy-r-BQ5BQRU6rzGS73-WBQ'
2024-12-01 02:51:30,305 - ERROR - URL Error https://60ed5b6d0df78.streamlock.net:1936/telemax/telemax/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='60ed5b6d0df78.streamlock.net', port=1936): Max retries exceeded with url: /telemax/telemax/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '60ed5b6d0df78.streamlock.net' doesn't match 's5.mexside.net'")))
2024-12-01 02:51:30,953 - ERROR - URL Error http://cm.hostlagarto.com:8081/Telemilenio/Telemilenio.myStream/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='cm.hostlagarto.com', port=8081): Max retries exceeded with url: /Telemilenio/Telemilenio.myStream/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf0a0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:51:47,027 - ERROR - URL Error https://cdn4.hostlagarto.com:8081/static/teleradionortetv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdn4.hostlagarto.com', port=8081): Max retries exceeded with url: /static/teleradionortetv/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce620>, 'Connection to cdn4.hostlagarto.com timed out. (connect timeout=15)'))
2024-12-01 02:52:02,316 - ERROR - URL Error http://37.187.7.106/teleribera/live.m3u8: HTTPConnectionPool(host='37.187.7.106', port=80): Max retries exceeded with url: /teleribera/live.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc9a0>, 'Connection to 37.187.7.106 timed out. (connect timeout=15)'))
2024-12-01 02:52:06,865 - ERROR - URL Error https://5d16127744872.streamlock.net/TVUNO/TVUNO/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='5d16127744872.streamlock.net', port=443): Max retries exceeded with url: /TVUNO/TVUNO/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '5d16127744872.streamlock.net' doesn't match either of '5eac7b031d945.streamlock.net', 'www.5eac7b031d945.streamlock.net'")))
2024-12-01 02:52:07,510 - ERROR - URL Error http://cm.hostlagarto.com:8081/ESTV/estv.myStream/playlist.m3u8?fluxustv.m3u8&PlaylistM3UCL: HTTPConnectionPool(host='cm.hostlagarto.com', port=8081): Max retries exceeded with url: /ESTV/estv.myStream/playlist.m3u8?fluxustv.m3u8&PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcccd0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:52:23,296 - ERROR - URL Error http://185.29.68.24/tvb.m3u8: HTTPConnectionPool(host='185.29.68.24', port=80): Max retries exceeded with url: /tvb.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbce350>, 'Connection to 185.29.68.24 timed out. (connect timeout=15)'))
2024-12-01 02:52:24,981 - ERROR - URL Error https://terengganufmmobile.secureswiftcontent.com/rtm-ch033/rtm-ch033/chunklist.m3u8: HTTPSConnectionPool(host='terengganufmmobile.secureswiftcontent.com', port=443): Max retries exceeded with url: /rtm-ch033/rtm-ch033/chunklist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:52:25,436 - ERROR - URL Error https://live-edge01.telecentro.net.ar/live/smil:trm.smil/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='live-edge01.telecentro.net.ar', port=443): Max retries exceeded with url: /live/smil:trm.smil/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))
2024-12-01 02:52:25,436 - ERROR - URL Error plugin://plugin.video.youtube/play/?video_id=9k5PsJEnXKc: No connection adapters were found for 'plugin://plugin.video.youtube/play/?video_id=9k5PsJEnXKc'
2024-12-01 02:52:27,189 - ERROR - URL Error https://tc-live1.sanook.com/live/22302_ch5.m3u8: HTTPSConnectionPool(host='tc-live1.sanook.com', port=443): Max retries exceeded with url: /live/22302_ch5.m3u8 (Caused by SSLError(CertificateError("hostname 'tc-live1.sanook.com' doesn't match either of '*.liveplay.myqcloud.com', 'liveplay.myqcloud.com'")))
2024-12-01 02:52:28,179 - ERROR - URL Error http://live.thairath.co.th/trtv2/playlist.m3u8: HTTPConnectionPool(host='live.thairath.co.th', port=80): Max retries exceeded with url: /trtv2/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bd16680>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:52:33,521 - ERROR - URL Error https://video.streamingchilenos.cl:1936/8048/8048/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='video.streamingchilenos.cl', port=1936): Max retries exceeded with url: /8048/8048/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd16a10>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:52:34,006 - ERROR - URL Error https://cast.cdnseguro.com:19360/8092/8092.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cast.cdnseguro.com', port=19360): Max retries exceeded with url: /8092/8092.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf880>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:52:35,500 - ERROR - URL Error https://stream.oursnetworktv.com/latin/SRTC/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='stream.oursnetworktv.com', port=443): Max retries exceeded with url: /latin/SRTC/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname 'stream.oursnetworktv.com' doesn't match either of '*.ontvmx.com', 'ontvmx.com'")))
2024-12-01 02:52:50,516 - ERROR - URL Error http://212.104.160.156:1935/live/trebujenatv2/master.m3u8: HTTPConnectionPool(host='212.104.160.156', port=1935): Max retries exceeded with url: /live/trebujenatv2/master.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcca30>, 'Connection to 212.104.160.156 timed out. (connect timeout=15)'))
2024-12-01 02:52:51,875 - ERROR - URL Error https://tv-trtworld.live.trt.com.tr/master_720.m3u8: HTTPSConnectionPool(host='tv-trtworld.live.trt.com.tr', port=443): Max retries exceeded with url: /master_720.m3u8 (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1007)')))
2024-12-01 02:52:58,966 - ERROR - URL Error https://cdn-canalpaulo.ciclano.io:1443/canalpaulo/canalpaulo/playlist.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='cdn-canalpaulo.ciclano.io', port=1443): Max retries exceeded with url: /canalpaulo/canalpaulo/playlist.m3u8?ROGERIOTORRES (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf970>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:53:13,975 - ERROR - URL Error http://179.1.7.198:1935/live/TVAgroHD.stream_160p/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='179.1.7.198', port=1935): Max retries exceeded with url: /live/TVAgroHD.stream_160p/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcddb0>, 'Connection to 179.1.7.198 timed out. (connect timeout=15)'))
2024-12-01 02:53:15,522 - ERROR - URL Error https://livefocamundo.com:8081/tvaracati/index.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='livefocamundo.com', port=8081): Max retries exceeded with url: /tvaracati/index.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcece0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:53:15,652 - ERROR - URL Error https://59f2354c05961.streamlock.net:1443/tvdasartes/_definst_/tvdasartes/playlist.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='59f2354c05961.streamlock.net', port=1443): Max retries exceeded with url: /tvdasartes/_definst_/tvdasartes/playlist.m3u8?ROGERIOTORRES (Caused by SSLError(CertificateError("hostname '59f2354c05961.streamlock.net' doesn't match 'video03.logicahost.com.br'")))
2024-12-01 02:53:38,709 - ERROR - URL Error http://209.222.17.241:5080/LiveApp/streams/872258137935361699746456.m3u8?token=undefined: HTTPConnectionPool(host='209.222.17.241', port=5080): Max retries exceeded with url: /LiveApp/streams/872258137935361699746456.m3u8?token=undefined (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc2b0>, 'Connection to 209.222.17.241 timed out. (connect timeout=15)'))
2024-12-01 02:53:40,156 - ERROR - URL Error https://radiocret.net:8082/hls/tvcret.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='radiocret.net', port=8082): Max retries exceeded with url: /hls/tvcret.m3u8?PlaylistM3UCL (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)')))
2024-12-01 02:53:40,954 - ERROR - URL Error http://ventdelnord.tv:8080/girona/directe.m3u8: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:53:41,081 - ERROR - URL Error https://59f2354c05961.streamlock.net:1443/pascoal/_definst_/pascoal/playlist-480p.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59f2354c05961.streamlock.net', port=1443): Max retries exceeded with url: /pascoal/_definst_/pascoal/playlist-480p.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '59f2354c05961.streamlock.net' doesn't match 'video03.logicahost.com.br'")))
2024-12-01 02:53:42,568 - ERROR - URL Error https://59f2354c05961.streamlock.net:1443/tvdigitalbirigui/_definst_/tvdigitalbirigui/playlist-480p.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59f2354c05961.streamlock.net', port=1443): Max retries exceeded with url: /tvdigitalbirigui/_definst_/tvdigitalbirigui/playlist-480p.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '59f2354c05961.streamlock.net' doesn't match 'video03.logicahost.com.br'")))
2024-12-01 02:53:57,972 - ERROR - URL Error https://5f593df7851db.streamlock.net/evangelizar/tv/playlist.m3u8: HTTPSConnectionPool(host='5f593df7851db.streamlock.net', port=443): Max retries exceeded with url: /evangelizar/tv/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce860>, 'Connection to 5f593df7851db.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:54:12,996 - ERROR - URL Error https://5f593df7851db.streamlock.net/evangelizar/tv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='5f593df7851db.streamlock.net', port=443): Max retries exceeded with url: /evangelizar/tv/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce2f0>, 'Connection to 5f593df7851db.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:54:21,334 - ERROR - URL Error https://vdo1.streamgato.us:3359/live/tvexitoslive.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='vdo1.streamgato.us', port=3359): Max retries exceeded with url: /live/tvexitoslive.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcce20>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:54:21,700 - ERROR - URL Error http://midia.faap.br:80/faaplatamlive-live/stream_720/livestream.m3u8: HTTPConnectionPool(host='midia.faap.br', port=80): Max retries exceeded with url: /faaplatamlive-live/stream_720/livestream.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbce830>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:54:21,928 - ERROR - URL Error http://midia.faap.br/faaplatamlive-live/stream_480/livestream.m3u8: HTTPConnectionPool(host='midia.faap.br', port=80): Max retries exceeded with url: /faaplatamlive-live/stream_480/livestream.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf040>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:54:22,294 - ERROR - URL Error http://midia.faap.br/faaplatamlive-live/stream_720/playlist.m3u8: HTTPConnectionPool(host='midia.faap.br', port=80): Max retries exceeded with url: /faaplatamlive-live/stream_720/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdba0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:54:22,515 - ERROR - URL Error http://midia.faap.br/faaplatamlive-live/stream_480/playlist.m3u8: HTTPConnectionPool(host='midia.faap.br', port=80): Max retries exceeded with url: /faaplatamlive-live/stream_480/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf130>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:54:23,167 - ERROR - URL Error https://europa-crtvg.flumotion.com/crtvg/europa_high/playlist.m3u8: HTTPSConnectionPool(host='europa-crtvg.flumotion.com', port=443): Max retries exceeded with url: /crtvg/europa_high/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf130>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:23,501 - ERROR - URL Error http://europa-crtvg.flumotion.com/playlist.m3u8: HTTPConnectionPool(host='europa-crtvg.flumotion.com', port=80): Max retries exceeded with url: /playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcc3d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:23,879 - ERROR - URL Error http://america-crtvg.flumotion.com/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='america-crtvg.flumotion.com', port=80): Max retries exceeded with url: /playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf940>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:24,322 - ERROR - URL Error http://europa-crtvg.flumotion.com/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='europa-crtvg.flumotion.com', port=80): Max retries exceeded with url: /playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcead0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:24,554 - ERROR - URL Error https://streaming01.zas.media/gideoes/programacao/playlist.m3u8: HTTPSConnectionPool(host='streaming01.zas.media', port=443): Max retries exceeded with url: /gideoes/programacao/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf760>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:54:30,071 - ERROR - URL Error https://livefocamundo.com:8081/tvjsid/index.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='livefocamundo.com', port=8081): Max retries exceeded with url: /tvjsid/index.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd6f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:55,161 - ERROR - URL Error https://cdnh4.iblups.com/hls/RMuwrdk7M9.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdnh4.iblups.com', port=443): Max retries exceeded with url: /hls/RMuwrdk7M9.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce0b0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:55,281 - ERROR - URL Error http://cdnh4.iblups.com/hls/irtp.m3u8: HTTPConnectionPool(host='cdnh4.iblups.com', port=80): Max retries exceeded with url: /hls/irtp.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf4c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:54:55,538 - ERROR - URL Error http://rtmp.cdn.upx.net.br/723b/myStream.sdp/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='rtmp.cdn.upx.net.br', port=80): Max retries exceeded with url: /723b/myStream.sdp/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcdf30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))
2024-12-01 02:55:11,718 - ERROR - URL Error http://www.portalcultura.com.br/playerhtml/funtelpa/tv_funtelpa/playlist.m3u8: HTTPConnectionPool(host='www.portalcultura.com.br', port=80): Max retries exceeded with url: /playerhtml/funtelpa/tv_funtelpa/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd6f0>, 'Connection to www.portalcultura.com.br timed out. (connect timeout=15)'))
2024-12-01 02:55:12,175 - ERROR - URL Error https://srv.panelcast.net/puruwalive/puruwalive/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='srv.panelcast.net', port=443): Max retries exceeded with url: /puruwalive/puruwalive/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd16470>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:55:31,689 - ERROR - URL Error http://live.cdn.upx.com/00046/myStream.sdp/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='live.cdn.upx.com', port=80): Max retries exceeded with url: /00046/myStream.sdp/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bd17be0>, 'Connection to live.cdn.upx.com timed out. (connect timeout=15)'))
2024-12-01 02:55:36,040 - ERROR - URL Error https://janus.tvturf.cl/playlist/stream.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='janus.tvturf.cl', port=443): Max retries exceeded with url: /playlist/stream.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce830>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:55:54,209 - ERROR - URL Error https://596639ebdd89b.streamlock.net:443/tvuniao/tvuniao/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='596639ebdd89b.streamlock.net', port=443): Max retries exceeded with url: /tvuniao/tvuniao/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbccf70>, 'Connection to 596639ebdd89b.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:56:09,239 - ERROR - URL Error http://live.cdn.upx.com/7550/myStream.sdp/playlist.m3u8: HTTPConnectionPool(host='live.cdn.upx.com', port=80): Max retries exceeded with url: /7550/myStream.sdp/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcea70>, 'Connection to live.cdn.upx.com timed out. (connect timeout=15)'))
2024-12-01 02:56:09,597 - ERROR - URL Error https://593b04c4c5670.streamlock.net/8068/8068/chunklist_w2070277572.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8068/8068/chunklist_w2070277572.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:56:24,625 - ERROR - URL Error https://596639ebdd89b.streamlock.net/8124/8124/playlist.m3u8: HTTPSConnectionPool(host='596639ebdd89b.streamlock.net', port=443): Max retries exceeded with url: /8124/8124/playlist.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce110>, 'Connection to 596639ebdd89b.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:56:26,493 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8014/8014/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8014/8014/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:56:42,535 - ERROR - URL Error http://stream2.ba.gov.br/hls-live/livepkgr/_definst_/irdeb/pgm-1.m3u8: HTTPConnectionPool(host='stream2.ba.gov.br', port=80): Max retries exceeded with url: /hls-live/livepkgr/_definst_/irdeb/pgm-1.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbce1a0>, 'Connection to stream2.ba.gov.br timed out. (connect timeout=15)'))
2024-12-01 02:56:57,540 - ERROR - URL Error http://stream2.ba.gov.br:80/hls-live/livepkgr/_definst_/irdeb/pgm-1.m3u8: HTTPConnectionPool(host='stream2.ba.gov.br', port=80): Max retries exceeded with url: /hls-live/livepkgr/_definst_/irdeb/pgm-1.m3u8 (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcd3c0>, 'Connection to stream2.ba.gov.br timed out. (connect timeout=15)'))
2024-12-01 02:57:01,748 - ERROR - URL Error https://59f2354c05961.streamlock.net:1443/tvideonews/_definst_/tvideonews/playlist.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='59f2354c05961.streamlock.net', port=1443): Max retries exceeded with url: /tvideonews/_definst_/tvideonews/playlist.m3u8?ROGERIOTORRES (Caused by SSLError(CertificateError("hostname '59f2354c05961.streamlock.net' doesn't match 'video03.logicahost.com.br'")))
2024-12-01 02:57:01,882 - ERROR - URL Error https://59f2354c05961.streamlock.net:1443/tvideonews/_definst_/tvideonews/chunklist.m3u8: HTTPSConnectionPool(host='59f2354c05961.streamlock.net', port=1443): Max retries exceeded with url: /tvideonews/_definst_/tvideonews/chunklist.m3u8 (Caused by SSLError(CertificateError("hostname '59f2354c05961.streamlock.net' doesn't match 'video03.logicahost.com.br'")))
2024-12-01 02:57:08,452 - ERROR - URL Error https://cdnh79.iblups.com/hls/RMuwrdk7M9.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdnh79.iblups.com', port=443): Max retries exceeded with url: /hls/RMuwrdk7M9.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17a00>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:57:11,729 - ERROR - URL Error https://vp01-tvviptv.delivery.fuugo.com/ngrp:vp01-TVV_IPTV-50e217f2_all/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='vp01-tvviptv.delivery.fuugo.com', port=443): Max retries exceeded with url: /ngrp:vp01-TVV_IPTV-50e217f2_all/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:57:33,110 - ERROR - URL Error https://59ce1298bfb98.streamlock.net/uniradiotv/uniradiotv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='59ce1298bfb98.streamlock.net', port=443): Max retries exceeded with url: /uniradiotv/uniradiotv/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17df0>, 'Connection to 59ce1298bfb98.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:57:36,997 - ERROR - URL Error http://xrxs.net/video/live-p12ariz-1164.m3u8: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
2024-12-01 02:57:53,441 - ERROR - URL Error https://cdn4.hostlagarto.com:8081/static/canal18hd/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='cdn4.hostlagarto.com', port=8081): Max retries exceeded with url: /static/canal18hd/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf9a0>, 'Connection to cdn4.hostlagarto.com timed out. (connect timeout=15)'))
2024-12-01 02:57:57,853 - ERROR - URL Error http://v2.tustreaming.cl:80/visionplustv/index.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='v2.tustreaming.cl', port=80): Max retries exceeded with url: /visionplustv/index.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf460>: Failed to establish a new connection: [Errno 111] Connection refused'))
2024-12-01 02:57:58,945 - ERROR - URL Error https://media1.mcr.iol.pt/livefm/vodafone/livefm/vodafone_128k/chunks.m3u8: HTTPSConnectionPool(host='media1.mcr.iol.pt', port=443): Max retries exceeded with url: /livefm/vodafone/livefm/vodafone_128k/chunks.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdcc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:00,431 - ERROR - URL Error https://tv7api2.tv.init7.net/api/live/?channel=e8d9c6e0-3691-4314-ac4a-b5636280d8d4: HTTPSConnectionPool(host='tv7api2.tv.init7.net', port=443): Max retries exceeded with url: /api/live/?channel=e8d9c6e0-3691-4314-ac4a-b5636280d8d4 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf400>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:20,082 - ERROR - URL Error https://streamingwowza.com:1936/wuanplus/wuanplus/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='streamingwowza.com', port=1936): Max retries exceeded with url: /wuanplus/wuanplus/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce770>, 'Connection to streamingwowza.com timed out. (connect timeout=15)'))
2024-12-01 02:58:20,529 - ERROR - URL Error http://infantil-crtvg.flumotion.com/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='infantil-crtvg.flumotion.com', port=80): Max retries exceeded with url: /playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d3bbcf670>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:37,538 - ERROR - URL Error https://614013e273786.streamlock.net:443/XtoTv/XtoTv/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='614013e273786.streamlock.net', port=443): Max retries exceeded with url: /XtoTv/XtoTv/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bd17b20>, 'Connection to 614013e273786.streamlock.net timed out. (connect timeout=15)'))
2024-12-01 02:58:37,888 - ERROR - URL Error https://admdn1.cdn.mangomolo.com/yastv/smil:yastv.stream.smil/playlist.m3u8: HTTPSConnectionPool(host='admdn1.cdn.mangomolo.com', port=443): Max retries exceeded with url: /yastv/smil:yastv.stream.smil/playlist.m3u8 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcdcc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:40,016 - ERROR - URL Error https://mediacpstreamchile.com:1936/latinosmedia-1/latinosmedia-1/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /latinosmedia-1/latinosmedia-1/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf640>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:40,337 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8008/8008/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8008/8008/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:58:40,374 - ERROR - URL Error https://mediacpstreamchile.com:1936/bajocero/bajocero/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /bajocero/bajocero/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd4e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:40,494 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8002/8002/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8002/8002/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:58:40,796 - ERROR - URL Error https://mediacpstreamchile.com:1936/evavision/evavision/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /evavision/evavision/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcec80>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:58:56,530 - ERROR - URL Error https://server1.oklanet.cl:1936/maximavideo1/maximavideo1/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='server1.oklanet.cl', port=1936): Max retries exceeded with url: /maximavideo1/maximavideo1/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcea10>, 'Connection to server1.oklanet.cl timed out. (connect timeout=15)'))
2024-12-01 02:59:11,547 - ERROR - URL Error http://149.56.17.92:1935/portalfoxmix/_definst_/portalfoxmix/playlist.m3u8?PlaylistM3UCL: HTTPConnectionPool(host='149.56.17.92', port=1935): Max retries exceeded with url: /portalfoxmix/_definst_/portalfoxmix/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f1d3bbcead0>, 'Connection to 149.56.17.92 timed out. (connect timeout=15)'))
2024-12-01 02:59:11,638 - ERROR - URL Error https://mediacpstreamchile.com:1936/ricardoaravena/ricardoaravena/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /ricardoaravena/ricardoaravena/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcf130>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:59:11,713 - ERROR - URL Error https://paneltv.net:3978/hybrid/play.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='paneltv.net', port=3978): Max retries exceeded with url: /hybrid/play.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcca60>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:59:27,047 - ERROR - URL Error https://zmlive.zappingtv.com/zm-free/zm.smil/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='zmlive.zappingtv.com', port=443): Max retries exceeded with url: /zm-free/zm.smil/playlist.m3u8?PlaylistM3UCL (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcef80>, 'Connection to zmlive.zappingtv.com timed out. (connect timeout=15)'))
2024-12-01 02:59:27,155 - ERROR - URL Error https://593b04c4c5670.streamlock.net:443/8038/8038/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='593b04c4c5670.streamlock.net', port=443): Max retries exceeded with url: /8038/8038/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '593b04c4c5670.streamlock.net' doesn't match either of 'presenze.stlex.it', 'www.presenze.stlex.it'")))
2024-12-01 02:59:27,186 - ERROR - URL Error https://mediacpstreamchile.com:1936/8102/8102/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /8102/8102/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbcd8a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 02:59:32,238 - ERROR - URL Error https://mediacpstreamchile.com:1936/8152/8152/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='mediacpstreamchile.com', port=1936): Max retries exceeded with url: /8152/8152/playlist.m3u8?PlaylistM3UCL (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1d3bbce080>: Failed to establish a new connection: [Errno -2] Name or service not known'))
2024-12-01 03:00:19,921 - ERROR - URL Error https://60ed5b6d0df78.streamlock.net:1936/canal28/canal28/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='60ed5b6d0df78.streamlock.net', port=1936): Max retries exceeded with url: /canal28/canal28/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '60ed5b6d0df78.streamlock.net' doesn't match 's5.mexside.net'")))
2024-12-01 03:00:32,374 - ERROR - URL Error https://5d16127744872.streamlock.net/COLOSAL/COLOSAL/playlist.m3u8?PlaylistM3UCL: HTTPSConnectionPool(host='5d16127744872.streamlock.net', port=443): Max retries exceeded with url: /COLOSAL/COLOSAL/playlist.m3u8?PlaylistM3UCL (Caused by SSLError(CertificateError("hostname '5d16127744872.streamlock.net' doesn't match either of '5eac7b031d945.streamlock.net', 'www.5eac7b031d945.streamlock.net'")))
2024-12-01 03:00:39,224 - ERROR - URL Error https://streaming03.zas.media:1936/tvfuturo/tvfuturo/playlist.m3u8?ROGERIOTORRES: HTTPSConnectionPool(host='streaming03.zas.media', port=1936): Max retries exceeded with url: /tvfuturo/tvfuturo/playlist.m3u8?ROGERIOTORRES (Caused by SSLError(CertificateError("hostname 'streaming03.zas.media' doesn't match 'tv02.zas.media'")))
2024-12-01 03:01:02,116 - debug - URL Streams https://www.youtube.com/channel/UCs231K71Bnu5295_x0MB5Pg/live: {}
2024-12-01 03:01:02,116 - debug - URL Streams https://www.youtube.com/channel/UCs231K71Bnu5295_x0MB5Pg/live: {}
2024-12-01 03:01:02,713 - debug - URL Streams https://www.youtube.com/user/TVPublicaArgentina/live: {}
2024-12-01 03:01:02,713 - debug - URL Streams https://www.youtube.com/user/TVPublicaArgentina/live: {}
2024-12-01 03:01:03,165 - debug - URL Streams https://www.youtube.com/channel/UCaU_cKHQ4IFHvBzQ-tKS1-Q/live: {}
2024-12-01 03:01:03,165 - debug - URL Streams https://www.youtube.com/channel/UCaU_cKHQ4IFHvBzQ-tKS1-Q/live: {}
2024-12-01 03:01:03,780 - debug - URL Streams https://www.youtube.com/c/cronicatv/live: {}
2024-12-01 03:01:03,780 - debug - URL Streams https://www.youtube.com/c/cronicatv/live: {}
2024-12-01 03:01:04,382 - debug - URL Streams https://www.youtube.com/user/c5n/live: {}
2024-12-01 03:01:04,382 - debug - URL Streams https://www.youtube.com/user/c5n/live: {}
2024-12-01 03:01:05,001 - debug - URL Streams https://www.youtube.com/c/americaenvivo/live: {}
2024-12-01 03:01:05,001 - debug - URL Streams https://www.youtube.com/c/americaenvivo/live: {}
2024-12-01 03:01:05,766 - debug - URL Streams https://www.youtube.com/channel/UCba3hpU7EFBSk817y9qZkiA/live: {}
2024-12-01 03:01:05,766 - debug - URL Streams https://www.youtube.com/channel/UCba3hpU7EFBSk817y9qZkiA/live: {}
2024-12-01 03:01:06,388 - debug - URL Streams https://www.youtube.com/channel/UC3OQ66wEJaOPHK25IBdWR_g/live: {}
2024-12-01 03:01:06,388 - debug - URL Streams https://www.youtube.com/channel/UC3OQ66wEJaOPHK25IBdWR_g/live: {}
2024-12-01 03:01:06,529 - debug - URL Streams https://www.youtube.com/channel/UCPr0HEFKN8wtDIfcPsVB10Q/live: {}
2024-12-01 03:01:06,529 - debug - URL Streams https://www.youtube.com/channel/UCPr0HEFKN8wtDIfcPsVB10Q/live: {}
2024-12-01 03:01:07,064 - debug - URL Streams https://www.youtube.com/channel/UCy-r-BQ5BQRU6rzGS73-WBQ/live: {}
2024-12-01 03:01:07,064 - debug - URL Streams https://www.youtube.com/channel/UCy-r-BQ5BQRU6rzGS73-WBQ/live: {}
2024-12-01 03:01:07,257 - error - URL Error No PluginError https://www.youtube.com/channel/UC1bBjOZieJWHbsFA0LwjjJA/Live: 
2024-12-01 03:01:07,257 - error - URL Error No PluginError https://www.youtube.com/channel/UC1bBjOZieJWHbsFA0LwjjJA/Live: 
2024-12-01 03:01:08,002 - debug - URL Streams https://www.youtube.com/channel/UCj6PcyLvpnIRT_2W_mwa9Aw/live: {}
2024-12-01 03:01:08,002 - debug - URL Streams https://www.youtube.com/channel/UCj6PcyLvpnIRT_2W_mwa9Aw/live: {}
2024-12-01 03:01:08,341 - debug - URL Streams https://www.youtube.com/user/encuentro/live: {}
2024-12-01 03:01:08,341 - debug - URL Streams https://www.youtube.com/user/encuentro/live: {}
2024-12-01 03:01:08,903 - debug - URL Streams https://www.youtube.com/c/TEC_ciencia_ar/live: {}
2024-12-01 03:01:08,903 - debug - URL Streams https://www.youtube.com/c/TEC_ciencia_ar/live: {}
2024-12-01 03:01:09,607 - debug - URL Streams https://www.youtube.com/channel/UCBQnpH3GKOKRGg4O8PpqQSw/live: {}
2024-12-01 03:01:09,607 - debug - URL Streams https://www.youtube.com/channel/UCBQnpH3GKOKRGg4O8PpqQSw/live: {}
2024-12-01 03:01:10,244 - debug - URL Streams https://www.youtube.com/channel/UCR9120YBAqMfntqgRTKmkjQ/live: {}
2024-12-01 03:01:10,244 - debug - URL Streams https://www.youtube.com/channel/UCR9120YBAqMfntqgRTKmkjQ/live: {}
2024-12-01 03:01:10,877 - debug - URL Streams https://www.youtube.com/channel/UC64ZNqX0FQHabP8iIkmnR3A/live: {}
2024-12-01 03:01:10,877 - debug - URL Streams https://www.youtube.com/channel/UC64ZNqX0FQHabP8iIkmnR3A/live: {}
2024-12-01 03:01:11,506 - debug - URL Streams https://www.youtube.com/channel/UC64ZNqX0FQHabP8iIkmnR3A/live: {}
2024-12-01 03:01:11,506 - debug - URL Streams https://www.youtube.com/channel/UC64ZNqX0FQHabP8iIkmnR3A/live: {}
2024-12-01 03:01:12,241 - debug - URL Streams https://www.youtube.com/channel/UC5UKMEIoqvNDMSDz2_6Sn9g/live: {}
2024-12-01 03:01:12,241 - debug - URL Streams https://www.youtube.com/channel/UC5UKMEIoqvNDMSDz2_6Sn9g/live: {}
2024-12-01 03:01:12,474 - debug - URL Streams https://www.youtube.com/channel/UCMy706Rhe2swaTmljCgiYVg/live: {}
2024-12-01 03:01:12,474 - debug - URL Streams https://www.youtube.com/channel/UCMy706Rhe2swaTmljCgiYVg/live: {}
2024-12-01 03:01:13,121 - debug - URL Streams https://www.youtube.com/c/canal26/live: {}
2024-12-01 03:01:13,121 - debug - URL Streams https://www.youtube.com/c/canal26/live: {}
2024-12-01 03:01:13,803 - debug - URL Streams https://www.youtube.com/c/CanalKZO/live: {}
2024-12-01 03:01:13,803 - debug - URL Streams https://www.youtube.com/c/CanalKZO/live: {}
2024-12-01 03:01:14,534 - debug - URL Streams https://www.youtube.com/c/canalciudadba/live: {}
2024-12-01 03:01:14,534 - debug - URL Streams https://www.youtube.com/c/canalciudadba/live: {}
2024-12-01 03:01:14,810 - debug - URL Streams https://www.youtube.com/user/diariodemocracia/live: {}
2024-12-01 03:01:14,810 - debug - URL Streams https://www.youtube.com/user/diariodemocracia/live: {}
2024-12-01 03:01:15,013 - debug - URL Streams https://www.youtube.com/channel/UC_7PDO6CCrynKPkFoOyLHOQ/live: {}
2024-12-01 03:01:15,013 - debug - URL Streams https://www.youtube.com/channel/UC_7PDO6CCrynKPkFoOyLHOQ/live: {}
2024-12-01 03:01:15,107 - error - URL Error https://www.youtube.com/c/showdelafe/live: Unable to open URL: https://www.youtube.com/c/showdelafe/live (404 Client Error: Not Found for url: https://www.youtube.com/c/showdelafe/live)
2024-12-01 03:01:15,107 - error - URL Error https://www.youtube.com/c/showdelafe/live: Unable to open URL: https://www.youtube.com/c/showdelafe/live (404 Client Error: Not Found for url: https://www.youtube.com/c/showdelafe/live)
2024-12-01 03:01:15,927 - debug - URL Streams https://www.youtube.com/c/canalciudadba/live: {}
2024-12-01 03:01:15,927 - debug - URL Streams https://www.youtube.com/c/canalciudadba/live: {}
2024-12-01 03:01:16,225 - error - URL Error No PluginError https://www.youtube.com/user/CanalPakapaka/LIVE: 
2024-12-01 03:01:16,225 - error - URL Error No PluginError https://www.youtube.com/user/CanalPakapaka/LIVE: 
2024-12-01 03:01:16,657 - error - URL Error No PluginError https://www.youtube.com/c/SenadoTVArgentina/LIVE: 
2024-12-01 03:01:16,657 - error - URL Error No PluginError https://www.youtube.com/c/SenadoTVArgentina/LIVE: 
2024-12-01 03:01:16,876 - debug - URL Streams https://www.youtube.com/channel/UC5KfW9_wv4tqbIPgY8tMG-w/live: {}
2024-12-01 03:01:16,876 - debug - URL Streams https://www.youtube.com/channel/UC5KfW9_wv4tqbIPgY8tMG-w/live: {}
2024-12-01 03:01:17,944 - debug - URL Streams https://www.youtube.com/channel/UC1h8_HKc5wScqeRw1HV6a6w/live: {}
2024-12-01 03:01:17,944 - debug - URL Streams https://www.youtube.com/channel/UC1h8_HKc5wScqeRw1HV6a6w/live: {}
2024-12-01 03:01:18,452 - debug - URL Streams https://www.youtube.com/channel/UCftan3cVkKKfxXpRN9JUX7A/live: {}
2024-12-01 03:01:18,452 - debug - URL Streams https://www.youtube.com/channel/UCftan3cVkKKfxXpRN9JUX7A/live: {}
2024-12-01 03:01:19,017 - debug - URL Streams https://www.youtube.com/c/SISEArgentina/live: {}
2024-12-01 03:01:19,017 - debug - URL Streams https://www.youtube.com/c/SISEArgentina/live: {}
2024-12-01 03:01:19,679 - debug - URL Streams https://www.youtube.com/channel/UCjeImUhRB1SLqN-svqDRJJw/live: {}
2024-12-01 03:01:19,679 - debug - URL Streams https://www.youtube.com/channel/UCjeImUhRB1SLqN-svqDRJJw/live: {}
2024-12-01 03:01:19,894 - debug - URL Streams https://www.youtube.com/channel/UCvCTWHCbBC0b9UIeLeNs8ug/live: {}
2024-12-01 03:01:19,894 - debug - URL Streams https://www.youtube.com/channel/UCvCTWHCbBC0b9UIeLeNs8ug/live: {}
2024-12-01 03:01:20,465 - debug - URL Streams https://www.youtube.com/channel/UCC1kfsMJko54AqxtcFECt-A/live: {}
2024-12-01 03:01:20,465 - debug - URL Streams https://www.youtube.com/channel/UCC1kfsMJko54AqxtcFECt-A/live: {}
2024-12-01 03:01:20,683 - debug - URL Streams https://www.youtube.com/channel/UC2zO2jZCDplwm8SVsXUcxCQ/live: {}
2024-12-01 03:01:20,683 - debug - URL Streams https://www.youtube.com/channel/UC2zO2jZCDplwm8SVsXUcxCQ/live: {}
2024-12-01 03:01:21,270 - debug - URL Streams https://www.youtube.com/channel/UCDSr0_1523BrSqQSm7Y4kHA/live: {}
2024-12-01 03:01:21,270 - debug - URL Streams https://www.youtube.com/channel/UCDSr0_1523BrSqQSm7Y4kHA/live: {}
2024-12-01 03:01:21,894 - debug - URL Streams https://www.youtube.com/c/canalciudadba/live: {}
2024-12-01 03:01:21,894 - debug - URL Streams https://www.youtube.com/c/canalciudadba/live: {}
2024-12-01 03:01:22,616 - debug - URL Streams https://www.youtube.com/c/Radio10am/live: {}
2024-12-01 03:01:22,616 - debug - URL Streams https://www.youtube.com/c/Radio10am/live: {}
2024-12-01 03:01:23,288 - debug - URL Streams https://www.youtube.com/channel/UCuJvDPQIaUhvdHiH275WjNA/live: {}
2024-12-01 03:01:23,288 - debug - URL Streams https://www.youtube.com/channel/UCuJvDPQIaUhvdHiH275WjNA/live: {}
